common/src/main/java/me/shedaniel/architectury/networking/NetworkManager.java

```diff
@@ -74,6 +75,19 @@ public static boolean canPlayerReceive(ServerPlayer player, ResourceLocation id)
         throw new AssertionError();
     }
     
+    /**
+     * Easy to use utility method to create a entity spawn packet.
+     * This packet is needed everytime any mod adds a entity.
+     * The entity should override {@link Entity#getAddEntityPacket()} to point to this method!
+     *
+     * @param entity The entity which should be spawned. The 'this' keyword should do the trick all the time.
```

Comment:
```suggestion
     * @param entity The entity which should be spawned.
```
Don't think we need to cater to the absolute beginners.